packages/dart/sshnoports/bin/sshnpd.dart

```diff
@@ -34,22 +26,33 @@ void main(List<String> args) async {
       ),
       usageCallback: (e, s) {
         printVersion();
-        stderr.writeln(SshnpdParams.parser.usage);
+        stderr.writeln(SshnpdOption.usage);
         stderr.writeln('\n$e');
       },
+      helpCallback: () {
+        printVersion();
+        stderr.writeln(SshnpdOption.usage);
+        exit(0);
+      },
```

Comment:
Added a help callback which is similar to usageCallback, but also exits and doesn't expect an error.